docs/usage/model_config.md

```diff
@@ -701,3 +701,100 @@ except ValidationError as e:
       Input should be a valid string [type=string_type]
     """
 ```
+
+## JSON schema customization
+
+#### `json_schema_serialization_defaults_required`
```

Comment:
There's a limit on characters. Also, I don't think we should use parameter name as the name of the section.

<img width="261" alt="Screenshot 2023-09-18 at 11 10 35" src="https://github.com/pydantic/pydantic/assets/7353520/38ba3670-ad9f-4e1c-b3e9-836575e5b87d">